spec/fixtures/cookbooks/oneview_test_api300_synergy/recipes/scope_change_resource_assignments.rb

```diff
@@ -0,0 +1,27 @@
+#
+# Cookbook Name:: oneview_test
+# Recipe:: scope_create
+#
+# (c) Copyright 2017 Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+#
+
+oneview_scope 'Scope-change_resource_assignments' do
+  api_version 300
```

Comment:
Hmm. I'm not sure why it's doing that for you. When I remove the api_version from these recipes it works fine for me.